extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php

```diff
@@ -1,41 +1,116 @@
-<nav class="global-navigation" id="globalNavigation">
-	<div class="global-navigation-container">
-		<div class="wikia-logo-container table-cell">
-			<a href="<?= htmlspecialchars( $centralUrl ) ?>"
-				class="wikia-logo"
-				rel="nofollow"
-				data-id="wikia-logo">
-				<img src="<?= $wg->BlankImgUrl ?>"
-					height="24"
-					width="91"
-					alt="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>"
-					title="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>">
-			</a>
-		</div>
-		<div class="hubs-container table-cell" id="hubsEntryPoint">
-			<?= $app->renderView( 'GlobalNavigation', 'hubsMenu' ) ?>
-		</div>
-		<div class="search-container table-cell">
-			<?= $app->renderView( 'GlobalNavigation', 'searchIndex' ); ?>
-		</div>
-		<div class="account-navigation-container table-cell">
-			<?= $app->renderView( 'GlobalNavigationAccountNavigation', 'index' ) ?>
-		</div>
-		<?php if ( !$isAnon && $notificationsEnabled ): ?>
+<?php if ( !$wg->EnableGlobalNav2016 ): ?>
+	<nav class="global-navigation" id="globalNavigation">
+		<div class="global-navigation-container">
+			<div class="wikia-logo-container table-cell">
+				<a href="<?= htmlspecialchars( $mainCorpPageUrl ) ?>"
+					class="wikia-logo"
+					rel="nofollow"
+					data-id="wikia-logo">
+					<img src="<?= $wg->BlankImgUrl ?>"
+						height="24"
+						width="91"
+						alt="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>"
+						title="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>">
+				</a>
+			</div>
+			<div class="hubs-container table-cell" id="hubsEntryPoint">
+				<?= $app->renderView( 'GlobalNavigation', 'hubsMenu' ) ?>
+	</div>
```

Comment:
[You must construct additional indents](http://knowyourmeme.com/memes/you-must-construct-additional-pylons). (starting from this line)